.github/workflows/create_release.yaml

```diff
@@ -0,0 +1,53 @@
+on:
+  push:
+    tags:
+      - 'v*'
+
+name: Create Release
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest # sorry
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get Previous Tag
+        run: |
+          PREV_TAG=$(git describe --abbrev=0 --tags "${{ github.ref }}^")
+          echo "::set-env name=baseRef::$PREV_TAG"
+
+      - name: Generate changelog
```

Comment:
cant we use an existing action that generates the changelog?

https://github.com/marketplace/actions/generate-changelog

Reply:
I skipped that option in general because it delegates the formatting to a third party. It's pretty easy to generate the content ourselves, and this gives us full control over the message appearance.

Though, I honestly didn't look at all the options out there. That specific one is surprisingly configurable and does some pretty neat stuff, so it might be worth reconsidering if we really want to use the same content here as in OBS. :)

Reply:
Changelog (actually _changelogs_) generation moved to external script.